.github/workflows/cross_platform_ci.yml

```
@@ -0,0 +1,102 @@
+name: rviz cross-platform RoboStack build
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel*"
+
+jobs:
+  run_rviz_compilation:
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Build Dependencies
+      uses: mamba-org/provision-with-micromamba@main
```

Shouldn't we pin the version here?
```suggestion
      uses: mamba-org/provision-with-micromamba@v11
```